package.json

```diff
@@ -24,7 +24,7 @@
   "version": "0.21.1",
   "private": true,
   "dependencies": {
-    "@hathor/wallet-lib": "^0.26.0",
+    "@hathor/wallet-lib": "^0.29.0",
```

Comment:
Is this version upgrade safe?

I don't remember what changed from version 0.26.0 to 0.29.0, maybe only wallet-service related stuff, but I think it is worth checking if any breaking changes happened

Reply:
From what I could read on [the releases page](https://github.com/HathorNetwork/hathor-wallet-lib/releases) and on [the difference between the codes](https://github.com/HathorNetwork/hathor-wallet-lib/compare/v0.26.0...v0.29.1) I could find no breaking change notification. Also, no change to dependencies.

All tests made locally on the wallet were successful.
Do you have any pointers to edge cases that may have been broken and must be tested?

Reply:
I don't, if you think this is safe, we can run the QA before merging to master